pkg/metrics/metrics.jsx

```diff
@@ -20,6 +20,7 @@
 import cockpit from 'cockpit';
 import React from 'react';
 import moment from "moment";
+import '../lib/patternfly/patternfly-cockpit.scss';
```

Comment:
This is rather nasty -- we'd import the whole PF3 just because of that install dialog. Can we port that to PF4 somehow?

Reply:
I can take a look. Maybe @KKoukiou can help me as well :) I'll ask her as she did some migration before.